### ISSUE TITLE
Rook Orchestrator module fails to initialize on Nautilus

### ISSUE THREAD

**Issue by travisn** [rook/rook#2263]
**Is this a bug report or feature request?**
* Bug Report

**Deviation from expected behavior:**
With the merge of #2187, the rook orchestrator module is enabled when configuring a Nautilus cluster. However, the module fails to be enabled.

**Expected behavior:**
The rook orchestrator module should succeed the configuration.

**How to reproduce it (minimal and precise):**
The operator log shows the following failure.
```
2018-11-02 22:23:05.925971 I | exec: Running command: ceph mgr module enable orchestrator_cli --force --cluster=rook-ceph --conf=/var/lib/rook/rook-ceph/rook-ceph.config --keyring=/var/lib/rook/rook-ceph/client.admin.keyring --format json --out-file /tmp/840770141
2018-11-02 22:23:06.933865 I | exec: Running command: ceph mgr module enable rook --force --cluster=rook-ceph --conf=/var/lib/rook/rook-ceph/rook-ceph.config --keyring=/var/lib/rook/rook-ceph/client.admin.keyring --format json --out-file /tmp/573222936
2018-11-02 22:23:07.978248 I | exec: Running command: ceph orchestrator set backend rook --cluster=rook-ceph --conf=/var/lib/rook/rook-ceph/rook-ceph.config --keyring=/var/lib/rook/rook-ceph/client.admin.keyring --format json --out-file /tmp/130807959
2018-11-02 22:23:08.420617 I | exec: no valid command found; 10 closest matches:
config-key rm <key>
config-key del <key>
config-key put <key> {<val>}
config-key set <key> {<val>}
config-key get <key>
osd tier add-cache <poolname> <poolname> <int[0-]>
osd tier remove-overlay <poolname>
osd tier rm-overlay <poolname>
osd tier set-overlay <poolname> <poolname>
osd tier cache-mode <poolname> none|writeback|forward|readonly|readforward|proxy|readproxy {--yes-i-really-mean-it}
Error EINVAL: invalid command
2018-11-02 22:23:08.421087 E | op-mgr: failed to enable orchestrator modules. failed to set rook as the orchestrator backend. exit status 22
```

**Comment by sebastian-philipp** [comment]
so, this is basically fixed, but needs to be verify with the Ceph n release

**Comment by travisn** [comment]
With nautilus as the base image (included in #2633), I see the nautilus modules loading correctly now. Since that's about to merge I'll go ahead and close this.
